src/vss_tools/vspec/vssexporters/vss2samm/helpers/stringHelper.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2024 Contributors to COVESA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+
+
+def str_to_lc_first(string_to_update: str):
+    if len(string_to_update) > 0:
+        return f"{string_to_update[0].lower()}{string_to_update[1:]}"
+    else:
+        return string_to_update
+
+
+def str_to_uc_first(string_to_update: str):
```

Comment:
Done.
I removed the str_to_uc_first and replaced its usage with .capitalize()

Reply:
Actually, I just revised the code and I will have to put back our implementation, because the capitalize function will convert:

asdf to Asdf

and aSdF **also** to Asdf.

In our case, the **string_to_update** can be inCamelCase and we want to preserve this CamelCase, so as result we need to have: **InCamelCase**